src/js/api/users-api.js

```diff
@@ -34,6 +36,14 @@ const Api = {
         .set('Content-Type', 'application/json')
         .send(body)
         .end((err, res) => {
+          if (err && !res) {
+            var errorResponse2 = {
```

Comment:
that duplicates what happens in the other case `(err || !res.ok)` - is there any particular reason not to adjust the existing condition that I might be missing?

Reply:
> that duplicates what happens in the other case `(err || !res.ok)` - is there any particular reason not to adjust the existing condition that I might be missing?

I am sure you are not missing anything.